src/main/webapp/content/scss/quiz.scss

```diff
@@ -1,87 +1,15 @@
-@import '~bootstrap/scss/bootstrap-grid.scss';
+@import './_artemis-mixins.scss';
+
+/* ==========================================================================
+    Quiz styles
+========================================================================== */
 .edit-quiz {
```

Comment:
Why does the quiz.scss still exist?

Reply:
Thanks for the review and the alternative to the ::ng-deep selector. I implemented that already. 
I have removed the styles which are only applicable to a single component from the quiz.scss and kept the styles that are used for several ones to avoid having to much duplicated styles. But if moving them to component style sheets is better to maintain, I can do that.